moxin-frontend/src/shared/mod.rs

```diff
@@ -6,3 +9,14 @@ pub mod resource_imports;
 pub mod styles;
 pub mod utils;
 pub mod widgets;
+
+pub fn live_design(cx: &mut Cx) {
```

Comment:
I would discuss this kind of changes first, and then apply them for everything if we wanna do them.

Reply:
I think I brought this up before, it should be a good idea.
If we agree on the change, I wouldn't delete this now, but for next time I would do all of these in a separate PR.

Reply:
I was reading the comments in order and since I saw @joulei thumbs up I applied the change for the rest of occurrences but I didn't saw this comment.

Shall I revert https://github.com/project-robius/moxin/pull/83/commits/d52c1ecdddedef424e33c380250685d36df8e469?